### PR TITLE
Removed unused prowjob scenario docs.

### PR DESCRIPTION
**Pull request by dekiel**
Removed limited tags on pr images notice.

**Related issue(s)**
See #10415

### CODE REVIEW

**Review by IwonaLanger — APPROVED** [review]

**Review by Sawthis — CHANGES_REQUESTED** [review]
This PR contains only formatting changes.

**Comment by dekiel** [comment]
Added missing changes. Added more details about default tags generated by the tool. Users duplicates tags by adding custom tags with value the same as default tag.

**Review by Sawthis — DISMISSED** [review]

File: cmd/image-builder/README.md [hunk 1]
```diff
@@ -119,6 +116,29 @@ See the accepted inputs description in the [image-builder reusable workflow](/.g
 The Image Builder reusable workflow provides outputs to pass the results of the build process.
 See the provided outputs description in the [image-builder reusable workflow](/.github/workflows/image-builder.yml) file.
 
+## Default Tags
+
+Image Builder provides a default tag for built images.
+The default tag is computed based on the template provided in the Image Builder configuration file.
+The default tag is always added to the image, even if the user provides custom tags.
+The Image Builder supports two default tags:
```
Comment (IwonaLanger) [inline cmd/image-builder/README.md:124]:
```suggestion
Image Builder supports two default tags:
```

File: cmd/image-builder/README.md [hunk 2]
```diff
@@ -119,6 +116,29 @@ See the accepted inputs description in the [image-builder reusable workflow](/.g
...
+
+## Named Tags
+
+Image Builder supports passing the name along with the tag, using both the `-tag` option and the config for the tag template.
+You can use `-tag name=value` to pass the name for the tag.
+
+If the name is not provided, it is evaluated from the value:
+
+- if the value is a string, it is used as a name directly. For example,`-tag latest` is equal to `-tag latest=latest`
+- if the value is go-template, it will be converted to a valid name. For example, `-tag v{{ .ShortSHA }}-{{ .Date }}` is equal
```
Comment (IwonaLanger) [inline cmd/image-builder/README.md:139]:
```suggestion
- If the value is a string, it is used directly as a name. For example,`-tag latest` is equal to `-tag latest=latest`
- If the value is go-template, it is converted to a valid name. For example, `-tag v{{ .ShortSHA }}-{{ .Date }}` is equal
```

**Review by Sawthis — APPROVED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**